src/main/scala/com/twitter/finagle/postgres/PostgresClient.scala

```diff
@@ -83,6 +83,7 @@ object PostgresClient {
     Types.TID -> TypeSpecifier("tidrecv", "tid"),
     Types.XID -> TypeSpecifier("xidrecv", "xid"),
     Types.CID -> TypeSpecifier("cidrecv", "cid"),
+    Types.JSON -> TypeSpecifier("json_recv", "json"),
```

Comment:
JSON was added as a baked-in feature in Postgres 9.2 (released in 2012, so it's pretty likely that most people are on _at least_ that version at this point).